elixir/apps/domain/lib/domain/policies/policy.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Domain.Policies.Policy do
+  use Domain, :schema
+  import Ecto.Changeset
+
+  schema "policies" do
+    field :name, :string
+
+    belongs_to :actor_group, Domain.Actors.Group
+    belongs_to :resource, Domain.Resources.Resource
+    belongs_to :account, Domain.Accounts.Account
+
+    field :created_by, Ecto.Enum, values: ~w[identity]a
+    belongs_to :created_by_identity, Domain.Auth.Identity
+
+    field :deleted_at, :utc_datetime_usec
+    timestamps()
+  end
+
+  @doc false
+  def changeset(policy, attrs) do
```

Comment:
You don't need this changeset here

Reply:
Whoops!  Thanks for the the heads up.  I kept telling myself to remove that once I created the `changeset.ex` file, but forgot

Reply:
This has been removed.